src/components/AppProvider/README.md

```diff
@@ -350,11 +350,12 @@ class ProviderThemeExample extends React.Component {
 
 ## Initializing the Shopify App Bridge
 
-When using Polaris, you don’t need to go through the initialization of the Shopify App Bridge as described [in the docs](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app). Instead, configure the connection to the Shopify admin through the app provider component, which must wrap all components in an embedded app. This component initializes the Shopify App Bridge using the `apiKey` you provide. **The `apiKey` attribute is required** and can be found in the [Shopify Partner Dashboard](https://partners.shopify.com).
+When using Polaris, you don’t need to go through the initialization of the Shopify App Bridge as described [in the docs](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app). Instead, configure the connection to the Shopify admin through the app provider component, which must wrap all components in an embedded app. This component initializes the Shopify App Bridge using the `apiKey` and `shopOrigin` you provide. **The `apiKey` attribute is required** and can be found in the [Shopify Partner Dashboard](https://partners.shopify.com).
+**The `shopOrigin` attribute is optional**, however it is best practice to [get and store `shopOrigin` yourself](https://help.shopify.com/en/api/embedded-apps/shop-origin).
```

Comment:
- Question about this! Is it a best practice to include the `shopOrigin` attribute? Or just that, if you DO include it, that you get and store it manually?

Reply:
```suggestion
**The `shopOrigin` attribute is optional, but recommended**.  Learn how to get and store the shop origin in the [Shopify Help Center](https://help.shopify.com/en/api/embedded-apps/shop-origin).
```

Reply:
i think we should not even mention that it's optional, because it really isn't (despite the props contract)